src/main/java/graphql/language/VariableDefinition.java

```diff
@@ -50,7 +53,22 @@ protected VariableDefinition(String name,
     public VariableDefinition(String name,
                               Type type,
                               Value defaultValue) {
-        this(name, type, defaultValue, null, new ArrayList<>(), IgnoredChars.EMPTY, emptyMap());
+        this(name, type, defaultValue, null, null, new ArrayList<>(), IgnoredChars.EMPTY, emptyMap());
+    }
+
+    /**
+     * alternative to using a Builder for convenience
+     *
+     * @param name         of the variable
+     * @param type         of the variable
+     * @param defaultValue of the variable
+     * @param directives   of the variable
+     */
+    public VariableDefinition(String name,
```

Comment:
we should not add any new constructors, but force new code to use the builder

Reply:
thanks for you advice, I have delete this new constructor.